src/CSModule.sol

```diff
@@ -741,6 +740,47 @@ contract CSModule is
                 pubkey
             );
 
+            bytes32 noPublicKeyPacked = _nodeOperatorPublicKeyPacked(
+                withdrawalInfo.nodeOperatorId,
+                pubkey
+            );
+            ExitPenaltyInfo storage exitPenaltyInfo = _delayedExitPenaltyInfo[
+                noPublicKeyPacked
+            ];
+            if (exitPenaltyInfo.penaltyValue != 0) {
+                accounting.penalize(
+                    withdrawalInfo.nodeOperatorId,
+                    exitPenaltyInfo.penaltyValue
+                );
+                emit DelayedValidatorExitPenalized(
+                    withdrawalInfo.nodeOperatorId,
+                    exitPenaltyInfo.penaltyValue
+                );
+                exitPenaltyInfo.penaltyValue = 0;
```

Comment:
This will result in an inconsistent result in the view function below

Reply:
explain pls

Reply:
If the penaltyValue is Zero isValidatorExitDelayPenaltyApplicable will return true